G/ginkgo/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ginkgo"
+version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/youwuyou/ginkgo.git", "c4128a21b1114bedfc19153a9508b2bd3b54954f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ginkgo/
+mkdir build && cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=/workspace/destdir \
```

Comment:
```suggestion
    -DCMAKE_INSTALL_PREFIX=${prefix} \
```

Reply:
It unfortunately did not work out well with ${prefix}, so we switched to the absolute path. A `usr/local` prefix would be added when using prefix and it seems like some packages underlying BinaryBuilder is managing this?

Reply:
It definitely should work with `${prefix}`, because that is what all other packages are using for this. Can you push this change to here so we can see the build logs?